src/javascript/app/pages/trade/analysis.js

```diff
@@ -89,14 +111,63 @@ const TradingAnalysis = (() => {
                 showExplanation();
             }
         }
+        if (current_tab) {
+            const el_to_show = getElementById(current_tab);
+            slideSelector(tab_selector_id, el_to_show);
+            const mobile_tab_header = document.getElementById('tab_mobile_header');
+            if (mobile_tab_header) mobile_tab_header.innerHTML = el_to_show.firstChild.innerHTML;
+        }
+        // workaround for underline during window resize
+        window.addEventListener('resize', () => {
+            slideSelector(tab_selector_id, getElementById(current_tab));
+        });
+    };
+
+    const slideSelector = (selector, el_to_show) => {
+        const isActive = el_to_show.classList.contains('active');
+        if (isActive) {
+            getElementById(`${selector}_selector`).setAttribute('style', `width: ${el_to_show.offsetWidth}px; margin-left: ${el_to_show.offsetLeft}px;`);
+        }
+    };
+
+    const goLeft = (e) => {
+        changeTab({ selector: e.target.getAttribute('data-parent'), direction: 'left' });
+    };
+
+    const goRight = (e) => {
+        changeTab({ selector: e.target.getAttribute('data-parent'), direction: 'right' });
+    };
```

Comment:
I'm not sure if the three above functions belong here. they seem better suited for a common file and then imported here.